getting-started/dcr-guide.rst

```diff
@@ -0,0 +1,102 @@
+######################################
+Generating DCR Coadds with LSST ComCam
+######################################
+
+A walkthrough for building differential chromatic refraction (DCR) coadds by running the
+`dcrAssembleCoaddTask` on an example set of ComCam image data. Prior to generating DCR coadds, you must
+have image templates.
+
+.. note::
+
+   This guide assumes the user has access to a shared Butler repository containing data from LSST ComCam via
+   the `US Data Facility (USDF) <https://developer.lsst.io/usdf/storage.html>`__. This guide further assumes
+   the user has a recently-built version of ``lsst.distrib`` from the `LSST Science Pipelines
+   <https://developer.lsst.io/usdf/stack.html>`__ (circa ``w_2022_30`` or later).
+
+Generating AP Templates
+=======================
+For detailed instructions on generating AP Templates, please see the `Building good seeing templates
+<https://pipelines.lsst.io/getting-started/dc2-guide.html>`__ guide for details.
+
+
+Assembling DCR Coadds
+=====================
+To generate DCR coadds, you must use the `dcrAssembleCoaddTask` found in the `drp_tasks` package. This will
+require image templates. Note: You will need to register your DCR subfilters prior to creating DCR coadds by
+using the following code:
+
+.. code-block:: python
+    butler register-dcr-subfilters /repo/main 3 'g'
+
+
+Creating Your DCR Assemble Coadd Pipeline
+-----------------------------------------
+The first step in generating DCR coadds is to make a pipeline that runs the `dcrAssembleCoaddTask` from the
+LSST 'drp_tasks' package. A sample `dcrAssembleCoadd` pipeline is shown below.
+
+.. code-block:: python
+
+    :name: dcrAssembleCoadd.yaml
```

Comment:
```suggestion
```

Reply:
Ope, looks like my explanation didn't save. I've noticed that these `:name:` things don't actually show up as expected on the guides. Based on a section in the DC2 guide, at the end of the applicable section they have something like this: "Save the BPS configuration file as `ApPipe-DC2-bps.yaml`."

So maybe: "Save the pipeline file as `dcrAssembleCoadd.yaml`." at the end of the code block?